lib/fs.js

```diff
@@ -196,10 +196,10 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this.atime = new Date(atim_msec);
-  this.mtime = new Date(mtim_msec);
-  this.ctime = new Date(ctim_msec);
-  this.birthtime = new Date(birthtim_msec);
+  this.atime = new Date(atim_msec + 0.5);
```

Comment:
Mind placing a comment above these. I can see someone doing a refactor a year from now and accidentally removing this.